src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/AccessorBuilder.cs

```diff
@@ -20,9 +20,9 @@ internal static class FastInvokerBuilder
         private delegate void StructSetDelegate<T, TArg>(ref T obj, TArg value);
         private delegate TResult StructGetDelegate<T, out TResult>(ref T obj);
 
-        private static MethodInfo s_createGetterInternal = typeof(FastInvokerBuilder).GetMethod(nameof(CreateGetterInternal), BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static);
-        private static MethodInfo s_createSetterInternal = typeof(FastInvokerBuilder).GetMethod(nameof(CreateSetterInternal), BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static);
-        private static MethodInfo s_make = typeof(FastInvokerBuilder).GetMethod(nameof(Make), BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static);
+        private static MethodInfo s_createGetterInternal = typeof(FastInvokerBuilder).GetTypeInfo().GetDeclaredMethod(nameof(CreateGetterInternal));
+        private static MethodInfo s_createSetterInternal = typeof(FastInvokerBuilder).GetTypeInfo().GetDeclaredMethod(nameof(CreateSetterInternal));
+        private static MethodInfo s_make = typeof(FastInvokerBuilder).GetTypeInfo().GetDeclaredMethod(nameof(Make));
```

Comment:
From [MSDN](https://msdn.microsoft.com/en-us/library/system.reflection.typeinfo.getdeclaredmethod(v=vs.110).aspx), `GetDeclaredMethod` will only return public method. Will this be a problem since the original code has `NonPublic` binding?